src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function Header({ headerTitle }) {
```

Comment:
Maybe rename to just `title`, imho no need to namespace with the component name.